### PR TITLE
Split project into 2 different sub-folders

### PR DESCRIPTION
**Pull request by samuelarbibe**
The goal is to create a better folder structure, that will make it easier to setup, run and develop on the project.  
  
The project is now divided into 2 different sub-folder, each with its own config files.  
To run the project, you can open both of these folders (and not the parent folder) into the same vscode workspace.
All required vscode launch commands are still available.  
  
  
### Code Changes  

1. `eslint` and `prettier-eslint` was installed. requires running `npm ci` in both folders, and installing the `prettier-eslint` vscode extension.
2. aliases were added into both of the projects. `../../components/library` -> `components/library`.
3. Server refactored, all content moved into `src` folder.
4. `server.js` refactored
5. DB connection util refactord, added default db to connection URI

### Workspace Changes
In summary, this PR requires you to:  

- Run `npm ci ` in both folders
- install the `prettier-eslint` VSCode extension
- rename `/?retryWrites=true&w=majority` -> `/eat?retryWrites=true&w=majority` in server `.env`

### CODE REVIEW

**Review by yotamberk — APPROVED** [review]
AMAZING! THANKS!
